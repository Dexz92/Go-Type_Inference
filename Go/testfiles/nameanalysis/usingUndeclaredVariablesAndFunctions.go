func main() {
  b := a;
  help();
}
