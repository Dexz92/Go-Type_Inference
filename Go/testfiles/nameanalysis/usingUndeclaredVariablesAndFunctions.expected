Error at line 2: symbol 'a' is not declared
Error at line 3: symbol 'help' is not declared
